src/components/navbar/Navbar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <nav class="flex items-center justify-between flex-wrap bg-stone-950 md:px-10 sm:px-6 py-6">
+    <div class="">
+      <RouterLink to="/"><img src="@/assets/HeimdalBanner_pos.png" alt="Heimdal Logo" class="h-24 object-contain" />
```

Comment:
Als we het design volgen denk ik dat het logo teveel naar links staat. Ook mogelijks wat te groot? (Header in het algemeen)